modules/olm-operatorgroups-rbac.adoc

```diff
@@ -11,13 +11,13 @@ When an Operator group is created, three cluster roles are generated. Each conta
 |===
 |Cluster role |Label to match
 
-|`<operatorgroup_name>-admin`
+|`olm.og.<operatorgroup_name>-admin-<hash-value>`
```

Comment:
<hash_value> ? Is it a user-replaced value? https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values